C#/Botkit/botbuilder-slack-adapter/DialogData.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace botbuilder_slack_adapter
```

Comment:
I think that we can use a namespace like `Microsoft.BotBuilder.SlackAdapter` or `BotBuilder.SlackAdapter`. We can discuss it later

Reply:
NOTE: we will update this in another PR.